fns/unbindService.js

```diff
@@ -26,11 +31,8 @@ module.exports = (api) => {
     }, asyncOperationInProgressCheck, (err, response, result) => {
       if (err && response && response.error_code === 'CF-ServiceBindingNotFound') {
```

Comment:
this block seems to be not needed but we assume there has been a problem that was mitigated like that.
we'd be glad to know what was happening there :)